content/1_General/Lambda_Funcs.mdx

```diff
@@ -50,22 +50,22 @@ description: 'Defining anonymous function objects.'
 
 ## Why Use Lambdas?
 Lambdas allow us to write simple and anonymous functions in place.
-This allows us to write smaller functions while keeping code local and 
+This allows us to write smaller functions while keeping code local and
 organized. In addition, lambdas can capture their surrounding variables, which
 is often convenient when writing helper functions.
 
-In addition, the C++ standard library has good support for lambdas, with functions 
-like `std::sort` and `std::lower_bound` allowing you to pass custom functions 
-to act as a comparator of objects. 
+In addition, the C++ standard library has good support for lambdas, with functions
+like `std::sort` and `std::lower_bound` allowing you to pass custom functions
+to act as a comparator of objects.
 
 ## General Form
 Lambdas have the following form:
 
-`[capture_list](parameters) -> trailing_return_type { 
-	// function body 
+`[capture_list](parameters) -> trailing_return_type {
```

Comment:
this doesn't render correctly. I assume you want to use triple backticks?

<img width="651" alt="Screenshot 2024-05-12 at 2 35 21 PM" src="https://github.com/cpinitiative/usaco-guide/assets/21228024/fc4403a8-6d2b-453b-9659-a388efb344f4">